support/monitoring/README.md

```diff
@@ -0,0 +1,117 @@
+# Setting of Monitoring for JASECI #
+
+
+## Prerequisite ##
+
+### Install Helm  ###
+
+https://helm.sh/docs/intro/install/
+
+
+
+## Promethues ##
+
+#### Step 1 ####
+
+##### add prometheus Helm repo  #####
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+
+#### Step 2 ####
+
+helm install prometheus prometheus-community/prometheus \
+    --set alertmanager.persistentVolume.storageClass="gp2" \
+    --set server.persistentVolume.storageClass="gp2"
+
+The Prometheus server can be accessed via port 80 on the following DNS name from within your cluster:
+prometheus-server.prometheus.svc.cluster.local
+
+#### Step 3 ####
+
+kubectl get all
+
+#### Step 4 ####
+
+Use Port-forward to test if promethues is running in your local browser
+
+kubectl port-forward -n prometheus deploy/prometheus-server 8080:9090
+
+
+
+
+## Grafana ##
+
+
+##### Go to grafana.yaml under grafana  Directory in the code and update the values of promethues URL as required #####
+
+
+#### Step 1 ####
+
+helm install grafana helmcharts/grafana \
+    --set persistence.storageClassName="gp2" \
```

Comment:
Does this only run on aws?

Reply:
if so we should make it clear that's the case and give a version of instructions to run this locally if that's possible.

Reply:
split in 2, one for local and one for AWS